pkg/azure/install.go

```diff
@@ -73,16 +73,17 @@ func (m *Mixin) Install() error {
 		outputs[newKey] = v
 	}
 
-	var lines []string
 	for _, output := range step.Outputs {
 		// ToUpper the key because of the case weirdness with ARM outputs
 		v, ok := outputs[strings.ToUpper(output.Key)]
 		if !ok {
 			return fmt.Errorf("couldn't find output key")
 		}
-		l := fmt.Sprintf("%s=%v", output.Name, v)
-		lines = append(lines, l)
+
+		err := m.Context.WriteMixinOutputToFile(output.Name, []byte(fmt.Sprintf("%v", v)))
```

Comment:
Could this be simplified to `[]byte(v)` like the other mixins? Or is there more going on there than I'm seeing?

Reply:
Yea, good question.  I didn't know quite what to do here, what with my unfamiliarity with the Azure API.  It appears to be a generic interface, so didn't know if `[]byte(v)` would error out in some cases?

Reply:
Okay reviewed the ARM code offline and this best preserves what was there before.